Formula/consul.rb

```diff
@@ -54,17 +54,22 @@ def install
     # Avoid running `go get`
     inreplace "GNUmakefile", "go get -u -v $(GOTOOLS)", ""
 
+    ENV["XC_OS"] = "darwin"
+    ENV["XC_ARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
     ENV["GOPATH"] = buildpath
     contents = Dir["{*,.git,.gitignore}"]
     (buildpath/"src/github.com/hashicorp/consul").install contents
 
     ENV.prepend_create_path "PATH", buildpath/"bin"
     Language::Go.stage_deps resources, buildpath/"src"
 
+    inreplace "src/github.com/hashicorp/go-bindata/go-bindata/main.go",
```

Comment:
@commitay this inreplace needs to be reported if you have a moment, though I suspect it will take care of itself before they release the next version :)

Reply:
I thought I had already comment on the consul `go-bindata` issue yesterday but apparently not. 

https://github.com/hashicorp/consul/issues/3868#issuecomment-364346730